### DISCUSSION TITLE
Is there a documentation repo?

### DISCUSSION THREAD

**Discussion by theodorekouzelis** [artilleryio/artillery#2668] [Artillery]
I've found a [broken link on the docs site](https://www.artillery.io/docs/get-started/why-artillery#:~:text=Artillery%20ships%20with%20several%20plugins). I can't seem to find the repo, so I can push a PR to fix it.

Are the docs open source?

**Comment by DominicABrooks** [comment]
Found some issues as well, commenting to bookmark this thread.

**Comment by hassy** [comment]
The docs aren't open source (yet). We'll fix that broken link @theodorekouzelis thank you! @DominicABrooks let us know what those issues are and we'll fix them as well. 🙏

**Reply by theodorekouzelis** [reply]
Awesome! Thank you

**Reply by bernardobridge** [reply]
Hi @theodorekouzelis , the links are fixed now! Thanks for reporting it.

@DominicABrooks let me know what those issues are, happy to fix them :)
